### DISCUSSION TITLE
The help document of juicefs on the mount command is inconsistent with the actual

### DISCUSSION THREAD

**Discussion by cs-lpc** [juicedata/juicefs#812] [General]
**What happened**:
When I use the mount command of juicefs to mount the directory according to the document, it cannot read the address of redis correctly.

**What you expected to happen**:
help document
<img width="558" alt="image" src="https://user-images.githubusercontent.com/90176164/132787486-8f866fa1-c73c-4f6f-907c-4575c6049e14.png">

use the command accroding the document
<img width="572" alt="image" src="https://user-images.githubusercontent.com/90176164/132786861-a2f80934-ff34-4545-ad72-0d7b7da6a9e5.png">

use the command accroding the code 
<img width="567" alt="image" src="https://user-images.githubusercontent.com/90176164/132787200-48699290-56e7-47a4-b1b0-aa05a9c342ec.png">

**How to reproduce it (as minimally and precisely as possible)**:
thie code simply read the first and second parameter as the Meta URL and mount point
<img width="613" alt="image" src="https://user-images.githubusercontent.com/90176164/132787006-3b821078-4b3a-4611-99e7-f2a0b1eae5f2.png">
**Anything else we need to know?**

**Environment**:
- JuiceFS version  0.17-dev

**Comment by davies** [comment]
There is a mistake in your first command line,  you should use `--no-syslog` without following `true`, or you need to use `=` to connect them, for example, `--no-syslog=true`.

I think the help is consistent.

**Comment by cs-lpc** [comment]
this confused me.
<img width="218" alt="image" src="https://user-images.githubusercontent.com/90176164/132796127-fcc6df67-1fd3-4e73-be83-24cb93b776e4.png">

**Comment by xiaogaozi** [comment]
@lipengchengcs May I ask which part of the confusion you are referring to?

**Comment by cs-lpc** [comment]
The usage of command options。The usage of `--get-timeout value` And `--no-syslog=true` is different

**Reply by xiaogaozi** [reply]
Basically, every option could use `--xxx=xxx` this format. The `--no-syslog` is a boolean type option, so if it present means its value is `true`, otherwise its value is `false`. The `--no-syslog=true` is an uncommon format.

**Reply by xiaogaozi** [reply]
We will update document to reduce user confusion.
